src/pages/media.js

```diff
@@ -13,10 +14,34 @@ export default () => {
         isLoading: isLoadingPodcasts,
     } = usePodcasts();
 
+    const [activeItem, setActiveItem] = useState('All');
+
+    const display = content => {
```

Comment:
Actually we can get away without `display` since it is just `setActiveItem`. Let's pass `setActiveItem` in as a prop for `handleClick` directly.